walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+@media screen and (min-width: 600px)
+{
+  .header-container
+  {
```

Comment:
again, I recommend hooking up an auto formatter

Reply:
yup, addressed in branch RCAVelez-redesignIndex, will propagate through further PRs